serverless/installing_serverless/serverless-install-config-options.adoc

```diff
@@ -0,0 +1,85 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-install-config-options"]
+= Advanced installation configuration options
+include::modules/common-attributes.adoc[]
+:context: serverless-install-config-options
+
+toc::[]
+
+This guide provides information for cluster administrators about advanced installation configuration options for Knative Serving and Knative Eventing.
+
+The YAML configuration will be shown by default when you are creating the Knative Serving or Knative Eventing custom resource definitions. However, you can access the form view by selecting *Edit Form* in the top right corner of the *Create Knative Serving* or *Create Knative Eventing* pages.
+
+[IMPORTANT]
+====
+Do not modify any YAML contained inside the `config` field. Some of the configuration values in this field are injected by the {ServerlessOperatorName}, and modifying them will cause your deployment to become unsupported.
+====
+
+== Knative Serving supported installation configuration options
+
+image::serving-form-view.png[Create Knative Serving form]
+
+=== Controller Custom Certs
+
+To enable tag-to-digest resolution, the Knative Serving controller requires access to the container registry.
+
+If your registry uses a self-signed certificate, you must enable tag-to-digest resolution by creating a ConfigMap or Secret.
+The {ServerlessOperatorName} will then automatically configure Knative Serving controller access to the registry.
+
+[IMPORTANT]
+====
+The ConfigMap or Secret must reside in the same namespace as the Knative Serving custom resource definition.
+====
+
+This example triggers the {ServerlessOperatorName} to:
+
+. Create and mount a volume containing the certificate in the controller.
+. Set the required environment variable properly.
+
+.Example YAML
+[source,yaml]
+----
```

Comment:
In other parts of docs we either always include the apiVersion and kind or we mention only the spec section. So, I'd probably include this here, to align with other examples:
```
apiVersion: operator.knative.dev/v1alpha1
kind: KnativeServing
```